machine/v1beta1/types_azureprovider.go

```diff
@@ -227,18 +292,33 @@ type DiskSettings struct {
 	EphemeralStorageLocation string `json:"ephemeralStorageLocation,omitempty"`
 }
 
-// ManagedDiskParameters is the parameters of a managed disk.
+// ManagedDiskParameters is the parameters of a OSDisk managed disk.
 type ManagedDiskParameters struct {
```

Comment:
You can rename the type as long you don't change any serialization.

Reply:
Ok, I was wondering that! Thanks :)
Renamed in https://github.com/openshift/api/pull/1119/commits/d2a93b66bad42fb7d316f51f802f5d22effd6ebc